cmd/hyperkube/kube-proxy.go

```diff
@@ -20,6 +20,7 @@ package main
 
 import (
 	kubeproxy "github.com/GoogleCloudPlatform/kubernetes/cmd/kube-proxy/app"
+	"github.com/GoogleCloudPlatform/kubernetes/contrib/mesos/pkg/hyperkube"
```

Comment:
yeah. They shouldn't be there for sure. Fixing this. Sorry for the inconvenience.